mm2src/mm2_tests.rs

```diff
@@ -1129,10 +1135,28 @@ async fn trade_base_rel_electrum(pairs: &[(&'static str, &'static str)]) {
     wait_log_re!(mm_alice, 22., ">>>>>>>>> DEX stats ");
 
     // Enable coins on Bob side. Print the replies in case we need the address.
-    let rc = enable_coins_eth_electrum(&mm_bob, &["http://195.201.0.6:8565"]).await;
+    let mut rc = enable_coins_eth_electrum(&mm_bob, &["http://195.201.0.6:8565"]).await;
+    rc.insert(
+        "tBTC",
+        enable_electrum(&mm_bob, "tBTC", false, &[
+            "electrum1.cipig.net:10068",
+            "electrum2.cipig.net:10068",
+            "electrum3.cipig.net:10068",
+        ])
+        .await,
+    );
     log! ({"enable_coins (bob): {:?}", rc});
```

Comment:
Trade tests fail on CI now as every agent has a custom-configured environment with a separate passphrase set. Please check CI logs and fill the addresses with tBTC to allow all trading tests to run through.